packages/fx-core/src/plugins/resource/frontend/utils/telemetry-helper.ts

```diff
@@ -12,6 +18,11 @@ export class telemetryHelper {
     measurements: { [key: string]: number } = {}
   ): void {
     properties[TelemetryKey.Component] = FrontendPluginInfo.PluginName;
+    properties[TelemetryKey.AppId] =
```

Comment:
Since there are duplicated code, is it possible to extract common logic as a method of 
```
static fillBasicTelemetry(properties, ctx) {
  properties[TelemetryKey.Component] = FrontendPluginInfo.PluginName;
  properties[TelemetryKey.AppId] =  (ctx.configOfOtherPlugins.get(DependentPluginInfo.SolutionPluginName)        ?.get(DependentPluginInfo.RemoteTeamsAppId) as string) ?? "";
}
```